.cirrus.yml

```diff
@@ -1001,6 +1016,8 @@ success_task:
         - buildah_bud_test
         - rootless_buildah_bud_test
         - upgrade_test
+        - image_build
```

Comment:
```suggestion
```

I think I yanked that task in https://github.com/containers/podman/pull/19780 (merged).  That might be what's breaking things.